singleFrameProcCoreTasks/charimg.rst

```diff
@@ -0,0 +1,304 @@
+#####################
+CharacterizeImageTask
+#####################
+
+Given an exposure with defects repaired (masked and interpolated over,
+e.g. as output by :doc:`IsrTask <isrtask>`), this task does initial
+source extraction and PSF estimation.
```

Comment:
Ok, done.  (I must have kept using the word  'defect' because it was in some of the initial descriptions i saw.)